starter/src/app/app.module.ts

```diff
@@ -81,6 +82,7 @@ import { NoContentComponent } from "./no-content";
 // load SASS styles
 import "../styles/styles.scss";
 /* tslint:enable */
+import "../styles/headings.css";
```

Comment:
What is this file for? It seems it doesn't exist... that is why the Travis job failed :s